core/field_variable.js

```diff
@@ -177,10 +164,14 @@ Blockly.FieldVariable.dropdownCreate = function() {
  */
 Blockly.FieldVariable.prototype.onItemSelected = function(menu, menuItem) {
   var itemText = menuItem.getValue();
```

Comment:
itemText feels overloaded now.  On line 166 (on the right) it's the ID, not the text.  But on line 172 it's the text.